core/src/com/unciv/models/ruleset/tile/TileImprovement.kt

```diff
@@ -31,8 +31,9 @@ class TileImprovement : NamedStats() {
         // todo UNIFY THESE
```

Comment:
While you're at it, why not take care of this tutu? It's only used by the Pyramids. Or maybe not - you're not yet touching Buildings.json.

Reply:
Fixed in #4115.